yellowbrick/text/postag.py

```diff
@@ -49,15 +51,12 @@ class PosTagVisualizer(TextVisualizer):
     only about a word’s definition, but also its use in context (for
     example the words “ship” and “shop” can be either a verb or a noun,
     depending on the context).
-
```

Comment:
Can I ask why these white spaces were removed? They seem important to the readability of the docstring - do you have some setting on your code editor that is removing them automatically? If so, would you mind turning that setting off and replacing these blank lines?

Reply:
I think that can be a problem with the editor because I didn't change the docstring for fit(). I will find a setting and get it right!